pleskdistup/actions/common_checks.py

```diff
@@ -94,6 +98,170 @@ def _do_check(self) -> bool:
         return False
 
 
+def get_known_php_versions() -> typing.List[str]:
+    # TODO: get rid of the explicit version list
+    return [version.PHPVersion(ver) for ver in (
+                "5.2", "5.3", "5.4", "5.5", "5.6",
+                "7.0", "7.1", "7.2", "7.3", "7.4",
+                "8.0", "8.1", "8.2", "8.3",
+                )
+            ]
+
+
+class AssertOutdatedPhpVersionNotInstalled(action.CheckAction):
```

Comment:
I think the issue with this name is that it doesn't explain why we're checking these versions. It seems like we just arbitrarily chose the minimum version. In other words, I'd prefer to include the word "outdated" in the name of the action to better reflect its purpose.

Reply:
I think, the name of an assertion should clearly state what it checks, not why it checks. "Why" is a matter of context. Like `fopen()` means you open a file, not why you open it.

It's OK to create a purpose-specific alias in a particular upgrader, e.g. `AssertOutdatedPhpVersionNotInstalled = AssertMinPhpVersionInstalled`, but dist-upgrader itself should be more generic.

It's also not clear what an assertion `AssertOutdatedPhpVersionNotInstalled(X)` checks, when you see it in code. Judging by its name, it fails if PHP X is installed (actually, it's not true currently). But will it fail if PHP X-1 is installed? Not clear from the name and since there's no docstring, you need to read the code to find that.

Reply:
However, since the behavior is a little different and I want to ensure the update of distupgraders goes smoothly, it seems necessary to verify the exact minimal version. Let's hope description of the action is well enough in this case.